test/distributed/cases/dml/select/values.result

```diff
@@ -0,0 +1,20 @@
+values row(1,1), row(2,2), row(3,3);
+column_0    column1
```

Comment:
Why there is an underline before 0, but not before 1?

Reply:
get it.  
typo was fixed